src/raft/raft.rs

```diff
@@ -619,6 +656,17 @@ impl<T: Storage> Raft<T> {
         info!("{} became candidate at term {}", self.tag, self.term);
     }
 
+    pub fn become_precandidate(&mut self) {
```

Comment:
s/become_precandidate/become_pre_candidate/
Just like it's always `PreCandidate` in the original PR.